downstream/modules/platform/ref-controller-system-requirements.adoc

```diff
@@ -45,19 +55,20 @@ h| Requirement | Required
 
 *Hop nodes*
 
-Serve to route traffic from one part of the automation mesh to another (for example, could be a bastion host into another network). RAM could affect throughput, CPU activity is low. Network bandwidth and latency are generally a more important factor than either RAM or CPU.
+Hop nodes serve to route traffic from one part of the automation mesh to another (for example, could be a bastion host into another network). RAM can affect throughput, CPU activity is low. Network bandwidth and latency are generally a more important factor than either RAM or CPU.
```

Comment:
"automation mesh" could be replaced with the associated attribute {AutomationMesh}